lib/find.coffee

```diff
@@ -55,6 +55,14 @@ module.exports =
       @findPanel.show()
       @findView.focusFindEditor()
 
+    @subscriptions.add atom.commands.add 'atom-workspace', 'find-and-replace:use-selection-as-replace-pattern', =>
+      return if @projectFindPanel?.isVisible() or @findPanel?.isVisible()
+
+      @createViews()
+      @projectFindPanel.hide()
```

Comment:
Sure. Should we do the same replacement for the
use-selection-as-find-pattern code above too?

On Fri, May 13, 2016, 8:38 PM Wliu <notifications@github.com> wrote:

> In lib/find.coffee
> <https://github.com/atom/find-and-replace/pull/719#discussion_r63183485>:
>
> > @@ -55,6 +55,14 @@ module.exports =
> >        @findPanel.show()
> >        @findView.focusFindEditor()
> >
> > +    @subscriptions.add atom.commands.add 'atom-workspace', 'find-and-replace:use-selection-as-replace-pattern', =>
> > +      return if @projectFindPanel?.isVisible() or @findPanel?.isVisible()
> > +
> > +      @createViews()
> > +      @projectFindPanel.hide()
>
> Can we use showPanel to replace this line and the two below it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/find-and-replace/pull/719/files/086b09a3b97c8f4bf3b82a826ecc719627b4e1f0#r63183485>
>